tests/integration/routes/test_error_page_templates.py

```diff
@@ -0,0 +1,233 @@
+from mock import Mock, patch
```

Comment:
We already have a `test_errors.py` integration test file, these should live there.